runbotics/runbotics-desktop/src/action/folder/folder.test.ts

```diff
@@ -288,4 +299,137 @@ describe('FolderActionHandler', () => {
+
+            removeTestFolder(`${TEMP_FOLDER_PATH}${path.sep}${TEST_FOLDER_NAME}`);
+        });
+    });
+
+    describe('Validate folder name', () => {
+        it('Should return true if name does not have illegal characters', () => {
+            const newName = 'Correct new folder name';
+
+            expect(() => folderActionHandler.checkIfNameIsValid(newName)).toBeTruthy();
+        });
+
+        it('Should throw error when folder name includes illegal character: \\', () => {
+            const newName = '\\';
+
+            expect(() => folderActionHandler.checkIfNameIsValid(newName)).toThrowError('Folder name cannot include the following characters: \\ / : * ? < > |');
+        });
+
+        it('Should throw error when folder name includes illegal character: /', () => {
+            const newName = '/';
+
+            expect(() => folderActionHandler.checkIfNameIsValid(newName)).toThrowError('Folder name cannot include the following characters: \\ / : * ? < > |');
+        });
+
+        it('Should throw error when folder name includes illegal character: :', () => {
+            const newName = ':';
+
+            expect(() => folderActionHandler.checkIfNameIsValid(newName)).toThrowError('Folder name cannot include the following characters: \\ / : * ? < > |');
+        });
+
+        it('Should throw error when folder name includes illegal character: *', () => {
+            const newName = '*';
+
+            expect(() => folderActionHandler.checkIfNameIsValid(newName)).toThrowError('Folder name cannot include the following characters: \\ / : * ? < > |');
+        });
+
+        it('Should throw error when folder name includes illegal character: ?', () => {
+            const newName = '?';
+
+            expect(() => folderActionHandler.checkIfNameIsValid(newName)).toThrowError('Folder name cannot include the following characters: \\ / : * ? < > |');
+        });
+
+        it('Should throw error when folder name includes illegal character: <', () => {
+            const newName = '<';
+
+            expect(() => folderActionHandler.checkIfNameIsValid(newName)).toThrowError('Folder name cannot include the following characters: \\ / : * ? < > |');
+        });
+
+        it('Should throw error when folder name includes illegal character: >', () => {
+            const newName = '>';
+
+            expect(() => folderActionHandler.checkIfNameIsValid(newName)).toThrowError('Folder name cannot include the following characters: \\ / : * ? < > |');
+        });
+
+        it('Should throw error when folder name includes illegal character: |', () => {
+            const newName = '|';
+
+            expect(() => folderActionHandler.checkIfNameIsValid(newName)).toThrowError('Folder name cannot include the following characters: \\ / : * ? < > |');
+        });
+    });
```

Comment:
maybe a one test that check for each of these unallowed chars? I'd be more compact. what do you think?